WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift

```diff
@@ -103,11 +113,35 @@ private extension TabbedTotalsCell {
         siteStatsDetailsDelegate?.tabbedTotalsCellUpdated?()
     }
 
+    func toggleNoResults() {
+        noResultsViewController.removeFromView()
+
+        let showNoResults = tabsData[filterTabBar.selectedIndex].dataRows.isEmpty && !limitRowsDisplayed
+        noResultsView.isHidden = !showNoResults
+
+        guard showNoResults,
+            let superview = superview else {
+                return
+        }
+
+        noResultsViewController.view.frame = noResultsView.frame
+        noResultsViewController.view.frame.origin.y = 0
```

Comment:
If we initialize line 127 with `bounds`, that should obviate the need for line 128.

Reply:
Nice one! Tis been changed! Thanks!